packages/components/src/custom-select-control-v2/legacy-component/index.tsx

```diff
@@ -64,12 +64,12 @@ function CustomSelectControl( props: LegacyCustomSelectProps ) {
 	const children = options.map(
 		( { name, key, __experimentalHint, style, className } ) => {
 			const withHint = (
-				<Styled.WithHintWrapper>
+				<Styled.WithHintItemWrapper>
 					<span>{ name }</span>
-					<Styled.ExperimentalHintItem className="components-custom-select-control__item-hint">
+					<Styled.WithHintItemHint className="components-custom-select-control__item-hint">
```

Comment:
`WithHintItemHint` sounds a bit odd. Does the hint item also need the `WithHint` prefix? Can't it just be `ItemHint` or something?

Reply:
It's a mouthful, I agree — although I personally like the clarity that the `WithHint` "namespace" brings when understanding the different styled components.